dream_layer_backend/test_api_standalone.py

```
@@ -0,0 +1,324 @@
+        
+        cls.server_thread = threading.Thread(target=run_server, daemon=True)
+        cls.server_thread.start()
+        
+        # Wait for server to start
+        time.sleep(2)
+        
+        # Check if server is available
+        try:
+            response = requests.get(cls.base_url, timeout=5)
+            cls.server_available = response.status_code == 200
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)